core/object/object.cpp

```diff
@@ -959,6 +959,10 @@ TypedArray<StringName> Object::_get_meta_list_bind() const {
 	return _metaret;
 }
 
+Vector<String> Object::_get_meta_list_bind_compat_76418() const {
+	return Variant(_get_meta_list_bind()).operator Vector<String>();
+}
+
```

Comment:
Implemented as suggested, tho because of `_bind_compatibility_methods` `object.compat.cpp` is a bit awkward (`Object::_bind_compatibility_methods` must always exist, or the `GDCLASS` macro would require #ifndefs inside itself as well)